src/AppRoutes.jsx

```diff
@@ -1,28 +1,22 @@
 // React Imports
 import React from 'react';
-import { Routes, Route, Navigate, Outlet } from 'react-router-dom';
+import { Routes, Route, Navigate } from 'react-router-dom';
 // Page Imports
-import Home from './routes/Home';
 import UserSection from './routes/UserSection';
 import Inbox from './routes/Inbox';
 import Forms from './routes/Forms';
 
-const ProtectedRoute = ({ isLoggedIn, children }) =>
-  isLoggedIn ? children ?? <Outlet /> : <Navigate to="PASS/" replace />;
-
-const AppRoutes = ({ isLoggedIn, loadingActive, loadMessages }) => {
+const AppRoutes = ({ loadingActive, loadMessages }) => {
   const restorePath = localStorage.getItem('restorePath');
   const path = restorePath ?? '/PASS/home';
 
   return (
     <Routes>
-      <Route exact path="/PASS/" element={isLoggedIn ? <Navigate to={path} replace /> : <Home />} />
-      <Route element={<ProtectedRoute isLoggedIn={isLoggedIn} />}>
```

Comment:
Able to get rid of this because the `Home` or `LandingPage` in `App.jsx` basically creates the protected route now correct?

Reply:
Pretty much, yeah

Reply:
Actually, this turned out to be an issue with the PR. Refactored it to remove these route changes